src/game/scenes/arena.c

```diff
@@ -929,42 +929,56 @@ void write_rec_move(scene *scene, game_player *player, int action) {
 
     memset(&move, 0, sizeof(move));
     move.tick = scene->gs->tick;
-    move.lookup_id = 2;
+    char *extra_data = sd_rec_set_lookup_id(&move, 2);
     move.player_id = 0;
-    move.action = 0;
 
     if(player == game_state_get_player(scene->gs, 1)) {
         move.player_id = 1;
     }
 
+    sd_action sda = 0;
     if(action & ACT_PUNCH) {
-        move.action |= SD_ACT_PUNCH;
+        sda |= SD_ACT_PUNCH;
     }
 
     if(action & ACT_KICK) {
-        move.action |= SD_ACT_KICK;
+        sda |= SD_ACT_KICK;
     }
 
-    if(action & ACT_UP) {
-        move.action |= SD_ACT_UP;
-    }
-
-    if(action & ACT_DOWN) {
-        move.action |= SD_ACT_DOWN;
-    }
-
-    if(action & ACT_LEFT) {
-        move.action |= SD_ACT_LEFT;
+    switch(action & ACT_Mask_Dirs) {
+        case ACT_UP:
```

Comment:
Not a blocker, but should we plan to unify the ACT_ and SD_ACT enums, so we don't have to convert between them?

Reply:
`SD_ACT_UPUP | SD_ACT_RIGHTRIGHT` does not equal `SD_ACT_UPRIGHT`.

I don't know how important being able to bit twiddle the openomf ACT_'s are to our game code offhand.

Anyhoo, unifying ACT_ & SD_ACT_ seems out of scope.